dataladConfig.sh

```diff
@@ -11,15 +11,19 @@ URL_RAW=''
 URL_DER=''
 
 # derivatives/preproc sibling url
-# for DEMO use: git@gin.g-node.org:/cpp_brewery/CPP_visMotion-cpp_spm-preproc.git
+# for DEMO use: git@gin.g-node.org:/cpp_brewery/CPP_visMotion-bidspm-preproc.git
 URL_DER_PREPROC=''
 
 # derivatives/stats sibling url
-# for DEMO use: git@gin.g-node.org:/cpp_brewery/CPP_visMotion-cpp_spm-stats.git
+# for DEMO use: git@gin.g-node.org:/cpp_brewery/CPP_visMotion-bidspm-stats.git
 URL_DER_STATS=''
 
+# GIN base name
+# to create siblings automatically for each new repo (i.e.  analysis_visualMotionLoc)
+GIN_BASENAME=''
+
 # set to "true" instead of "false" if you want to use the bleeding edge version of CPP SPM
-USE_CPP_SPM_DEV=false
+USE_BIDSPM_DEV=false
```

Comment:
I would probably nuke this and its use in the code as well.

Dev is most often lagging being main these days.